### PR TITLE
Custom logging level for infinity_polling

### PR DESCRIPTION
**Pull request by Badiboy**

### CODE REVIEW

File: telebot/__init__.py [hunk 1]
```diff
@@ -455,17 +455,28 @@ def __notify_update(self, new_messages):
         for listener in self.update_listener:
             self._exec_task(listener, new_messages)
 
-    def infinity_polling(self, timeout=20, long_polling_timeout=20, *args, **kwargs):
+    def infinity_polling(self, timeout=20, long_polling_timeout=20, logger_level=logging.ERROR, *args, **kwargs):
```
Comment (dannkunt) [inline telebot/__init__.py:458]: How should logging level be set?
What if somebody not using logging?
Maybe add type hinting for this

File: telebot/__init__.py [hunk 2]
```diff
@@ -455,17 +455,28 @@ def __notify_update(self, new_messages):
...
+        :param logger_level: Custom logging level for infinity_polling logging. None/NOTSET = no error logging
+        """
         while not self.__stop_polling.is_set():
             try:
                 self.polling(none_stop=True, timeout=timeout, long_polling_timeout=long_polling_timeout, *args, **kwargs)
             except Exception as e:
-                logger.error("Infinity polling exception: %s", str(e))
-                logger.debug("Exception traceback:\n%s", traceback.format_exc())
+                if logger_level and logger_level >= logging.ERROR:
+                    logger.error("Infinity polling exception: %s", str(e))
```
Comment (dannkunt) [inline telebot/__init__.py:471]: Why not f'string?

**Review by dannkunt — CHANGES_REQUESTED** [review]

File: telebot/__init__.py (hunk 1 above)
Comment (Badiboy) [reply telebot/__init__.py:458]: > How should logging level be set?
infinity_polling(..., logger_level=logging.INFO, ...)

> What if somebody not using logging?
¯\_(ツ)_/¯ Use logging...

> Maybe add type hinting for this
I'll add it to parameter description.

File: telebot/__init__.py (hunk 2 above)
Comment (Badiboy) [reply telebot/__init__.py:471]: https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/
(Not the best link, but I cannot find the best one fast)

**Pull request merged**